src/code/models/codap-connect.coffee

```diff
@@ -172,6 +184,52 @@ module.exports = class CodapConnect
       @codapPhone.call(getListing, doResolve)
       log.info "requested list of attributes"
 
+  _syncAttributeProperties: (attrProps, initialSync) ->
+    nodesToSync = if initialSync \
+                    then _.filter @graphStore.nodeKeys, (node) -> not node.codapID or not node.codapName \
+                    else _.map @graphStore.nodeKeys, (node) -> node
```

Comment:
This `(node) -> node` looked weird at first but then I realized the _.map callback is `(value, key)` so it works.  It really does look like a noop though.